scripts/qa_crowbarsetup.sh

```diff
@@ -28,6 +28,14 @@ distsuseip=$(dig -t A +short $distsuse)
 : ${want_raidtype:="raid1"}
 : ${want_multidnstest:=1}
 
+### want_${proposal} variables:
+# policy: empty or !=0 means its =1
+# defaults are evaluated in deploy_single_proposal()
+#
+# the following variables have different defaults
+: ${want_dvr:=0}
+: ${want_docker:=0}
```

Comment:
right, I guess I cleaned it up because its in the same namespace and wanted to behave consistently

I will do the cleanup in a separate commit and switch to use want_proposal_{proposalX}